data/human/news.txt

```diff
@@ -289,6 +289,7 @@ news "unemployed youth"
 			"Do you know how lucky you are, that you can visit a different world every week if you want to? I've been stuck on this one filthy little planet my whole life."
 			"If I had enough money, I'd go up to a captain in the spaceport and ask for transport to anywhere else. Nowhere's as bad as this place."
 			"Are you hiring new crewmembers? And are you by chance planning to fly to the Paradise Worlds?"
+			"Beat it, old-timer, I've got more important things to do than chat with you."
```

Comment:
```suggestion
			"Beat it, old-timer. I've got more important things to do than chat with you."
```
Another spot where you could let this slide if it sounds better with the comma. Your choice.